java/runtime/src/main/java/io/ray/runtime/AbstractRayRuntime.java

```diff
@@ -194,6 +180,11 @@ public PlacementGroup getPlacementGroup(PlacementGroupId id) {
     return gcsClient.getPlacementGroupInfo(id);
   }
 
+  @Override
+  public PlacementGroup getPlacementGroup(String name, boolean global) {
+    return gcsClient.getPlacementGroupInfo(name, global);
```

Comment:
Should be `getPlacementGroupInfo`? `AbstractRayRuntime#getPlacementGroup` just return whatever `gcsClient.getPlacementGroupInfo` returns.

Reply:
hm... I don't think so, the naming of `AbstractRayRuntime#getPlacementGroup` actually just follows the previous design like `AbstractRayRuntime#createActor` `AbstractRayRuntime#getActorHandle`